include/reactor-cpp/environment.hh

```diff
@@ -36,10 +37,12 @@ private:
   const log::NamedLogger log_;
   const unsigned int num_workers_{default_number_worker};
   unsigned int max_reaction_index_{default_max_reaction_index};
-  const bool run_forever_{default_run_forever};
+  bool run_forever_{default_run_forever};
   const bool fast_fwd_execution_{default_fast_fwd_execution};
 
   std::set<Reactor*> top_level_reactors_{};
+  /// Set of actions that act as an input to the reactor program in this environment
```

Comment:
Is this the set of physical actions then ?

Reply:
At the moment yes, but it will include enclave (and federated) connections soon.